src/main/java/org/jpeek/skeleton/OpsOf.java

```diff
@@ -87,4 +103,26 @@ public void visitMethodInsn(final int opcode,
             .up().up();
     }
 
+    /**
+     * Returns fully qualified name, an unambiguous name
+     * that specifies field without regard
+     * to the context of the call.
+     * @param owner The class the attribute belongs to
+     * @param attr The name of the field
+     * @return Fully qualified name of the field
+     */
+    private static String getQualifiedName(final String owner,
```

Comment:
@ilyakharlamov Let's extract it to a class, something like `public final class QualifiedName extends TextEnvelope`.